lstchain/mc/mc.py

```diff
@@ -1,29 +1,93 @@
 import numpy as np
 
 
-def power_law_integrated_distribution(emin, emax, tot_num_events, spectral_index, bin_number=30):
+def power_law_integrated_distribution(emin, emax, nev, sp_idx, bin_number=30):
```

Comment:
why changing the names to make them **less** explicit?

Reply:
Hi @vuillaut, thanks! I wanted to reduce variable names because they were becoming too long and not specially comfortable for all the subsequent calculations. In this case, I wanted to unify it with the other names in the same file. I can return these to the previous values and also expand the others if there is a strong opinion about this

Reply:
names returned to their previous more explicit ones

Reply:
I understand but I would unify in the other direction, making all names explicit (if we still want to produce code following pep8 and zen conventions).
When a user will use the function and sees `spectral_index`, it will seem more obvious than `sp_idx` without looking at the documentation.
A good editor with auto-completion should help when typing code using already defined functions.

Reply:
although I do not like it that much, I agree with you that your proposal goes more with the python philosophy. I'll open an issue to change this naming in the near future, is that ok?